src/app/wallets/get-lightning-fee.ts

```diff
@@ -9,6 +8,7 @@ import { CachedRouteLookupKeyFactory } from "@domain/routes/key-factory"
 import { checkedToWalletId } from "@domain/wallets"
 import { LndService } from "@services/lnd"
 import { RoutesCache } from "@services/redis/routes"
+import { getBalanceForWallet } from "./get-balance-for-wallet"
```

Comment:
I guess we should no longer import with `from "@app/wallets"` anymore after this refactoring, so that we always go through the index.ts at the root of apps?

Reply:
I tried to make it consistent:
- If the method is required outside the `app` (like in servers or tests) then `from "@app"` should be used
- If the method is required in a different submodule (inside `app`) then you can use  `from "@app/wallets"`
- If the method is required in the same submodule then we should use relative imports (this is to avoid multiple loads of the same resources, circular references, ...)